platform/linux-generic/odp_timer.c

```diff
@@ -1352,16 +1340,20 @@ int odp_timer_init_global(const odp_init_t *params)
 #else
 	ODP_DBG("Using lock-less timer implementation\n");
 #endif
+	conf_str =  "timer.inline";
+	if (!_odp_libconfig_lookup_int(conf_str, &val)) {
+		ODP_ERR("Config option '%s' not found.\n", conf_str);
+		return -1;
```

Comment:
You should back out the previous `odp_shm_reserve()` here before failing to avoid leaking `timer_global->shm`. Alternately, you could be a bit more forgiving and simply default to `use_inline_timers = false` if you can't find an override.

Reply:
Will fix.